programs/dmatest/run

```diff
@@ -0,0 +1,22 @@
+#!/bin/sh
+# - timeout
+# - iterations
+# - run
+
+# checks if dmatest is an existing module, and if it is, run it
+if modinfo dmatest | grep -q "ERROR: Module dmatest not found"; then
+    echo "ERROR: Module dmatest not found."
+    exit 1
+else
+    # clear the kernel log, since dmatest writes to it and we wanna see what it outputs
+    dmesg -c
```

Comment:
Thanks, usually we'd like to keep all dmesg since there could be other boot warnings before test run. And in order to analyze the dmatest results, one approach is to grep the needed info to output, like
```
dmesg | grep "qat_dev.* started .* acceleration engines"
dmesg | grep torture:
```

Then it requires programs/dmatest/parse to analyze the output to generate test stats, refer to programs/rcutorture/parse